src/Symfony/Component/Translation/Bridge/Phrase/README.md

```diff
@@ -0,0 +1,115 @@
+Not doing so will result in Phrase creating a new locale for the imported keys.
+
+Locale creation
+---------------
+
+If you define a locale in your `translation.yaml` which is not configured in your Phrase project, it will be automatically created. Deletion of locales however, is (currently) not managed by this provider.
+
+Domains as tags
+---------------
+
+Translations will be tagged in Phrase with the Symfony translation domain they belong to.
+Check the [wickedone/phrase-translation-bundle](https://github.com/wickedOne/phrase-translation-bundle) if you need help managing your tags in Phrase.
+
+Cache
+-----
+
+The read responses from Phrase are cached to speed up the read and delete method of this provider and also to contribute to the rate limit as little as possible.
+Therefor the factory should be initialised with a PSR-6 compatible cache adapter.
+
+Events
+------
+
+To enable you to perform post-processing on translation values and / or keys, two events are dispatched by this provider class.
+
+**PhraseReadEvent**
+
+_After_ reading the catalogue from Phrase, the resulting `TranslatorBag` is dispatched in a `PhraseReadEvent` prior to being returned from the read method.
+
+**PhraseWriteEvent**
+
+_Before_ writing the catalogue to Phrase, the `TranslatorBag` is dispatched in a `PhraseWriteEvent`.
+
+Fine tuning your Phrase api calls
+---------------------------------
+
+You can fine tune the read and write methods of this provider by adding query parameters to your dsn configuration.
+General usage is `read|write[option_name]=value`
+
+**example:** `phrase://PROJECT_ID:API_TOKEN@default?read[encoding]=UTF-8&write[update_descriptions]=0`
+
+See tables below for available options and, if applicable, their default values.
+
+**Read**
+
+In order to read translations from Phrase the [download locale](https://developers.phrase.com/api/#get-/projects/-project_id-/locales/-id-/download) call is made to the Phrase api. This call provides the following options.
+
+| name                              |  type  |  default value   |
+|-----------------------------------|:------:|:----------------:|
+| `branch`                          | string |                  |
+| `include_empty_translations`      |  bool  |        1         |
+| `exclude_empty_zero_forms`        |  bool  |                  |
+| `include_translated_keys`         |  bool  |                  |
+| `keep_notranslate_tags`           |  bool  |                  |
+| `format_options`                  | array  | enclose_in_cdata |
+| `encoding`                        | string |                  |
+| `skip_unverified_translations`    |  bool  |                  |
+| `include_unverified_translations` |  bool  |                  |
+| `use_last_reviewed_version`       |  bool  |                  |
+| `fallback_locale_enabled`         |  bool  |        0         |
```

Comment:
where is the doc for this one?

Reply:
this is an "internal" configuration option (which indeed wasn't really made clear) which forces the provider to use the fallback locales configured in phrase for the read operation. changed the README and added a bit more detail on that